pkg/postage/interface.go

```diff
@@ -11,9 +11,16 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
+// SyncStatus interface definitions reflect the status of the postage syncing progress.
+type SyncStatus interface {
+	Get() (bool, error)
```

Comment:
`GetSyncStatus` otherwise this is just a getter setter naming which is not very comprehensible